CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+
+- For in-line suggestions, use the [GitHub suggesting feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request). 
+- You can comment on one line or several lines. You can provide changes or edits, and the person who authored the PR can easily apply your change, or create their own competing suggestion, and you can discuss it. To invite others to the conversation, tag their GitHub @username in a comment to notify them. In this way, you can discuss any change. It’s transparent and inclusive.
+- The PR owner can merge in your suggested commits one at a time or in batch (preferred).
+- When you are providing a more granular extensive review that results a large number (more than 20) of in-line suggestions, work with the PR owner to develop a workflow. You might decide together that as a reviewer, you can check out the branch and make the changes yourself.
+
+## Pull Requests (PRs)
+
+Provide a helpful, informative PR title that include the context and page you are updating. Each PR must address one issue or feature.
+
+To provide a fix or update, make a direct contribution. If you're not a member or maintainer, fork the repo and then submit a pull request (PR) from your forked repo to the `main` branch.
+
+Start by creating a [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/). Create your draft PR early, even if your work is just beginning or incomplete. Your draft PR indicates to the community that you're working on something and provides a space for conversations early in the development process. Merging is blocked for `Draft` PRs, so they provide a safe place to experiment and invite comments.
+
+All pull requests must get code reviewed. As the PR submitter, you must address each comment and suggestion in the **Files changed** view during the review process.
+
+- If you are a member of the docs repo, you can merge the approved PR.
+- If your PR is a fork, the code maintainers will merge the approved PR.
+
+### Continuous Integration (CI)
+
+The docs output is generated by a CI pipeline which means that we can update the docs site whenever we want, so the [Mina Docs](https://docs.minaprotocol.com/) website always contains the most up-to-date content available. 
+
+During the documentation build process, each PR goes through Vercel CI to test the doc updates and build a preview of your changes. Vercel for GitHub automatically deploys and provides preview deployment URLs. After the PR status is ready, click **Visit Preview** and navigate to the changed page to see your doc updates. If you want to speed up the PR review process, you can add a link from the PR description to the updated preview deployment page URL.
```

Comment:
```suggestion
During the documentation build process, each PR goes through Vercel CI to test the doc updates and build a preview of your changes. Vercel for GitHub automatically deploys and provides preview deployment URLs. After the PR status is ready, click **Visit Preview** and navigate to the changed page to see your doc updates.
```
Not sure I understand the last sentence given the previous sentence is correct that the preview URL is generated automatically.